pkg/notify/notify_test.go

```diff
@@ -44,16 +48,38 @@ func (s *notifySuite) TestNotifyHub(c *check.C) {
 }
 
 func (s *notifySuite) TestContinusStop(c *check.C) {
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
```

Comment:
why this test case is changed?

Reply:
the 5000 goroutine started in original code is too slow in my computer